pkgs/tools/misc/atuin/default.nix

```diff
@@ -11,20 +11,20 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "atuin";
-  version = "15.0.0";
+  version = "16.0.0";
 
   src = fetchFromGitHub {
-    owner = "ellie";
+    owner = "atuinsh";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-BX1WpvJMcfpepsRX0U6FJBL5/+mpUyTZxm65BbbZLJA=";
+    hash = "sha256-Kh6aaWYV+ZG7Asvw5JdGsV+nxD+xvvQab5wLIedcQcQ=";
   };
 
   # TODO: unify this to one hash because updater do not support this
```

Comment:
No one has an idea how we can resolve this TODO?

Reply:
no idea unfortunately